client/app/bundles/server-components/ServerComponentsPage.jsx

```
@@ -0,0 +1,129 @@
+              <h3 className="font-medium text-slate-800 mb-1">Selective Hydration</h3>
+              <p>
+                Only client components (like the toggle above) receive JavaScript.
+                Everything else is pure HTML — zero hydration cost.
+              </p>
+            </div>
+          </div>
+        </section>
+      </div>
+    </div>
+  );
+};
```

ESLint

[error] Function component is not a function declaration

(react/function-component-definition)